openai.chat/Package.md

```diff
@@ -1 +1,67 @@
-# Client to connect to OpenAI
+Connects to the OpenAI Chat API from Ballerina with `ballerinax/openai.chat` package.
+
+## Package Overview
+The `ballerinax/openai.chat` package is a [Ballerina](https://ballerina.io/) connector for OpenAI's Chat API. The connector provides access to state-of-the-art models for Chat. By using the `ballerinax/openai.chat` package, you can easily integrate OpenAI's advanced AI capabilities into your own Ballerina applications.
+
+|                             | Version         |
+|-----------------------------|-----------------|
+| OpenAI API                  | 1.2.0           |
+
+## Prerequisites
+
+Before using this connector in your Ballerina application, complete the following:
+
+* Create an [OpenAI Account](https://beta.openai.com/signup/).
+* Obtain an API key by following [these instructions](https://platform.openai.com/docs/api-reference/authentication).
+
+## Quickstart
+
+To use the OpenAI Chat connector in your Ballerina application, update the `.bal` file as follows:
+
+### Step 1: Import connector
+First, import the `ballerinax/openai.chat` module into the Ballerina project.
+
+```ballerina
+import ballerinax/openai.chat;
+```
+
+### Step 2: Create a new connector instance
+Create and initialize `chat:Client` with the `apiKey` obtained.
+```ballerina
+    chat:Client chatClient = check new ({
+        auth: {
+            token: "sk-XXXXXXXXX"
+        }
+    });
+```
+
+### Step 3: Invoke connector operation
+1. Now you can use the operations available within the connector. Note that they are in the form of remote operations.
+
+    Following is an example on creating a conversation with the GPT-3.5 model:
+
+    Conversation with GPT-3.5-turbo
```

Comment:
```suggestion
    Conversation with GPT-3.5-turbo
```
Is this a sub-heading?

Reply:
https://github.com/hevayo/ballerina-openai-client/pull/10/files#r1138523502